src/components/SearchBox/__tests__/SearchBox-test.js

```diff
@@ -438,6 +438,20 @@ describe('SearchBox', () => {
       expect(mount(<SearchBox {...defaultProps} />)).toMatchSnapshot();
     });
 
+    test('sets autocorrect... properties', () => {
+      const res = render(
+        <SearchBox {...defaultProps} autofocus={true} query="sample query" />
+      );
+      const input = res.getByDisplayValue('sample query');
+
+      expect(input.getAttribute('autofocus')).toBe('true');
```

Comment:
We have helpers for DOM elements in this file. Did you try them?

```suggestion
      expect(input).toHaveAttribute('autofocus', 'true');
```

Reply:
did not know about those, thanks